test-three/amy-menu/src/menu.tsx

```diff
@@ -0,0 +1,93 @@
+import { Component } from "react";
+import menuJson from "./menu-data.json";
+
+type MenuItemProps = {
+  name: string;
+};
+
+class MenuSubItem extends Component<MenuItemProps> {
+  render() {
+    return (
+      <li>
+        {this.props.name}
+      </li>
+    );
+  }
+}
+
+class MenuItem extends Component<MenuItemProps> {
+  render() {
+    return (
+      <ul>
+        {this.props.name}
+        {this.props.children}
+      </ul>
+    );
+  }
+}
+
+export class Menu extends Component {
+  parseJsonFile() {
```

Comment:
is the file not already an object?

Reply:
I assumed the JSON file was a different object that needed parsing to change from that object to the JSON representation for JavaScript object properties; thanks for clarifying.